openconnect_sso/app.py

```diff
@@ -179,3 +181,10 @@ def run_openconnect(auth_info, host, proxy, args):
 
     logger.debug("Starting OpenConnect", command_line=command_line)
     return subprocess.run(command_line).returncode
+
+
+def handle_disconnect(command):
+    if command:
+        logger.info(f"Running {command!r} on shutdown...")
```

Comment:
Please use keyword arguments to template log messages, e.g `logger.info("Running command on disconnect", command_line=command)

Reply:
Will do. I'm new to `structlog`. Looks very nice.